packages/cadl-ranch-specs/http/azure/core/basic/main.tsp

```diff
@@ -206,6 +206,120 @@ Expected response body:
 """)
 op listWithPage is global.Azure.Core.Foundations.Operation<{}, global.Azure.Core.Page<User>>;
 
+model ListItemInput<Type> {
+  @doc("The body of the input.")
+  @body
+  bodyInput: ListItemInputBody;
+
+  @doc("Another query parameter.")
+  @query
+  another?: Type;
+}
+
+#suppress "@azure-tools/typespec-azure-core/use-extensible-enum" "For testing"
+@doc("A fixed enum input paramter.")
+@global.Azure.Core.fixed
+enum ListItemInputFixedEnum {
+  @doc("The first enum value.")
+  First,
+
+  @doc("The second enum value.")
+  Second,
+}
+
+@doc("An extensible enum input paramter.")
+enum ListItemInputExtensibleEnum {
+  @doc("The first enum value.")
+  First,
+
+  @doc("The second enum value.")
+  Second,
+}
+
+@doc("The body of the input.")
+model ListItemInputBody {
+  @doc("The name of the input.")
+  inputName: string;
+}
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing global.Azure.Core.Page"
+@scenario
+@doc("List with int32 parameter Azure.Core.Page<>.")
+@route("/int-page")
+@scenarioDoc("""
+Expected query parameter: api-version=2022-12-01-preview&another=1
+
+Expected body parameter: {"inputName": "Madge"}
```

Comment:
Why List have a body?

Reply:
We have real case from confidential ledger.

Reply:
It's a `@get`!

Reply:
Yes, it is a get and it has body. And it is a real case from confidential ledger.